packages/react/src/components/Button/Button.tsx

```diff
@@ -32,6 +32,11 @@ export type ButtonProps = {
    * @default false
    */
   asChild?: boolean;
+  /**
+   * Specify the type of button. Unset when `asChild` is true
+   * @default 'button'
+   */
+  type?: 'button' | 'submit' | 'reset';
 } & ButtonHTMLAttributes<HTMLButtonElement>;
```

Comment:
Should we omit type from `ButtonHTMLAttributes<HTMLButtonElement>`?

Reply:
Sure we could do that

Reply:
Yes, and still omit, to make sure we get our jsdoc